malaria24/ona/models.py

```diff
@@ -194,7 +198,8 @@ def get_digest_email_data(self, facility_code):
 
     def send_digest_email(self):
         for manager in Actor.objects.provincial():
-            context = self.get_digest_email_data(manager.facility_code)
+            context = self.get_digest_email_data(
+                manager.province, manager.facility_code)
```

Comment:
If context is `{}` (as the case for when neither the `province` or `faciliy_code` is specified) then this current code will send a blank email. I'd rather that no email gets sent at all and that you [log a warning](https://github.com/praekelt/malaria24-django/blob/develop/malaria24/ona/models.py#L541-L542) instead.

To not send an email, you can do that by doing:
```python
if not context:
    logging.warning('No province or facility_code for %s.' % manager.name)
    continue
```